src/ra_fwd.h

```diff
@@ -5,25 +5,37 @@
 // Forward declaring namespace std caused problems
 #include <xstring>
 
-
-namespace ra {
-
 #ifndef _TCHAR_DEFINED
 #if _MBCS
-using TCHAR = char;
+using TCHAR  = char;
+using PTCHAR = char*;
+using TBYTE  = unsigned char;
+using PTBYTE = unsigned char*;
```

Comment:
BYTE is always unsigned char (8-bits). Why do we need a TBYTE?

Reply:
A precaution. In WinNT.h.
<code>
#ifndef _TCHAR_DEFINED
typedef char TCHAR, *PTCHAR;
typedef unsigned char TBYTE , *PTBYTE ;
#define _TCHAR_DEFINED
#endif /* !_TCHAR_DEFINED */
<code>

<code>
#ifdef  UNICODE                     // r_winnt

#ifndef _TCHAR_DEFINED
typedef WCHAR TCHAR, *PTCHAR;
typedef WCHAR TBYTE , *PTBYTE ;
#define _TCHAR_DEFINED
#endif /* !_TCHAR_DEFINED */
...
<code>